dashboard/modules/metrics/grafana_dashboard_factory.py

```diff
@@ -0,0 +1,416 @@
+# flake8: noqa E501
+
+import copy
+import json
+import os
+from dataclasses import dataclass
+
+from typing import List
+
+
+@dataclass
+class Target:
+    expr: str
+    legend: str
+
+
+@dataclass
+class Panel:
```

Comment:
Can you write a simple docstring for Panel and Target? Also it'd be nice to mention the title here should match the file inside Metrics.tsx